app/uk/gov/hmrc/eusubsidycompliance/models/types/package.scala

```diff
@@ -60,7 +60,7 @@ package object types extends SimpleJson {
   type TraderRef = String @@ TraderRef.Tag
   object TraderRef
       extends RegexValidatedString(
-        regex = """.{0,35}"""
+        regex = """[A-Za-z0-9 ]{1,400}"""
```

Comment:
`\s` is the regex for white space, Ive asked Mas to update on the frontend. Can we do the same on the backend.

Reply:
please ignore the above comment. `\s` matches tabs as well as other whitespace chars.